eng/pipelines/templates/steps/create-apireview.yml

```diff
@@ -0,0 +1,25 @@
+parameters:
+  Artifacts: []
+
+steps:
+  - task: DownloadPipelineArtifact@2
```

Comment:
Depending on the context we may not need to explicitly download this. Also when we do this in the release side we might need to use the packages-signed artifact if we plan to store them anywhere on the server side.